deepchecks/checks/performance/simple_model_comparison.py

```diff
@@ -127,90 +180,89 @@ def _find_score(self, train_ds: Dataset, test_ds: Dataset, task_type: ModelType,
-        results = [simple_score, pred_score]
-        fig = go.Figure([go.Bar(x=models, y=results)])
-        fig.update_layout(width=600, height=500)
-        fig.update_yaxes(title=score_name)
-
-        return CheckResult({'given_model_score': pred_score,
-                            'simple_model_score': simple_score,
-                            'ratio': ratio},
-                           display=[text, fig])
-
-    def add_condition_ratio_not_less_than(self, min_allowed_ratio: float = 1.1):
+    def add_condition_ratio_not_less_than(self, min_allowed_ratio: float = 1.1, classes: List[Hashable] = None):
         """Add condition - require min allowed ratio between the given and the simple model.
 
         Args:
             min_allowed_ratio (float): Min allowed ratio between the given and the simple model -
             ratio is given model / simple model (if the scorer returns negative values we divide 1 by it)
+            classes (List[Hashable]): Used in multiclass models to limit condition only to given classes.
         """
-        def condition(result: Dict) -> ConditionResult:
-            ratio = result['ratio']
-            if ratio < min_allowed_ratio:
-                return ConditionResult(False,
-                                       f'The given model performs {_more_than_prefix_adder(ratio, self.maximum_ratio)} '
-                                       'times compared to the simple model using the given scorer')
+        def condition(result: Dict, max_ratio=self.maximum_ratio, class_list=classes) -> ConditionResult:
+            scores_df = result['scores']
+            task_type = result['type']
+            metrics = scores_df['Metric'].unique()
+
+            metrics_dfs = []
+            if task_type == ModelType.MULTICLASS:
+                if class_list is None:
+                    class_list = scores_df['Class'].unique()
+                for metric, clas in itertools.product(metrics, class_list):
+                    score_rows = scores_df[(scores_df['Metric'] == metric) & (scores_df['Class'] == clas)]
+                    metrics_dfs.append(score_rows)
+            else:
+                for metric in metrics:
+                    score_rows = scores_df[(scores_df['Metric'] == metric)]
+                    metrics_dfs.append(score_rows)
+
+            not_passing_metrics = set()
+            for df in metrics_dfs:
+                origin_score = df[df['Type'] == 'Origin']['Value'].iloc[0]
+                simple_score = df[df['Type'] == 'Simple']['Value'].iloc[0]
+                ratio = get_scores_ratio(simple_score, origin_score, max_ratio)
+                if ratio < min_allowed_ratio:
+                    not_passing_metrics.update(df['Metric'].tolist())
+
+            if not_passing_metrics:
+                not_passing_metrics = sorted(not_passing_metrics)
+                msg = f'Metrics with scores ratio lower than threshold: {", ".join(not_passing_metrics)}'
```

Comment:
Think we should report not passing classes.

Reply:
I think it will be too long message